bin/postinstall

```diff
@@ -6,6 +6,21 @@ set -e
 
 CLI="${APP_NAME}"
 
+# Check for custom gems
+custom_gemfile=$(${CLI} config:get CUSTOM_PLUGIN_GEMFILE || echo "")
+if [ -e "${custom_gemfile}"]; then
+
+	# Need to override the frozen setting for the vendored gems
+	${CLI} run bundle config --local frozen 0
+
+	# Re-bundle the application including gems
+	${CLI} run bundle install
```

Comment:
I couldn't find an official statement on that, but when I run `--no-deployment`, its ignoring the `vendor/cache` and installs _all_ dependencies.

`BUNDLE_FROZEN` simply says ignore the restriction on Gemfile.lock, but it only fetches all Git (not sure if we can avoid that) and my custom dependencies.